.github/workflows/build-preview.yaml

```diff
@@ -21,8 +21,8 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      # for debugging
-      - uses: crazy-max/ghaction-dump-context@v2
+      - name: Debug dump
```

Comment:
Gave this a name in several places, just to look a little nicer and more consistent.